src/components/input.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react';
+
+class Input extends Component {
+  constructor() {
+    super();
+    this.state = {value:''}
+    this.handleChange = this.handleChange.bind(this)
+  }
+
+  handleChange(e) {
+    this.setState({value: e.target.value})
```

Comment:
you are using the value as state of both this component and the parent component, avoid duplication